css/modals/video-quality/_video-quality.scss

```diff
@@ -1,16 +1,16 @@
 .video-quality-dialog {
     color: $modalTextColor;
 
-    .video-quality-dialog-title {
-        margin-bottom: 10px;
+    .hide-warning {
+        visibility: hidden;
     }
 
     .video-quality-dialog-contents {
         align-items: center;
         color: $modalTextColor;
         display: flex;
         flex-direction: column;
-        padding: 10px;
+        padding: 0 10px 10px;
```

Comment:
It looks a little bit too tight. I know that we cannot leave as much space because it will look too empty in the other case. So, if we can't adjust it in a way that it looks ok in both cases, we may add some default text like "Adjust your receive video quality" or something similar.
<img width="495" alt="screenshot 2017-08-14 16 07 06" src="https://user-images.githubusercontent.com/3264676/29291900-b5bd4ac8-810a-11e7-9519-f9f0fbb5cc53.png">

This is the previous one for comparison:
<img width="443" alt="screenshot 2017-08-14 16 08 16" src="https://user-images.githubusercontent.com/3264676/29291993-0a86f2de-810b-11e7-8f5c-6f2d8e17b425.png">